server/controllers/addFeedback.js

```diff
@@ -0,0 +1,22 @@
+const Airtable = require('airtable');
+
+const { Airtable_API_KEY } = process.env;
+const base = new Airtable({ apiKey: Airtable_API_KEY }).base('appAZnpLnWP0wjAc6');
+
+exports.addFeedback = (req, res) => {
+  const { userAirtableId } = req;
+  const { feedback } = req.body;
+
+  base('Users').update(
+    userAirtableId,
+    {
+      Feedback: feedback,
+    },
+    (error) => {
```

Comment:
This is very minor/pedantic but to keep consistent with the rest of the codebase, can we call error arguments `err`.